builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -9,6 +9,23 @@ steps:
   inputs:
     version: '3.1.x'
 
+- script: npm install -g azure-functions-core-tools@3 --unsafe-perm true
+  displayName: 'Install Azure Functions Core Tools'
+
+- task: DockerInstaller@0
+  displayName: Docker Installer
+  inputs:
+    dockerVersion: 17.09.0-ce
+    releaseType: stable
+        
+- script: docker pull mcr.microsoft.com/mssql/server:2019-latest
+  displayName: Pull MSSQL Docker Image
+
+- script: 'docker run -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=Placeholder1" -e "MSSQL_PID=Express"
```

Comment:
We should generally be avoiding hardcoding passwords. I'd suggest having the pipeline randomly generate one and pass that in instead.

Reply:
Added a step to generate password using the BuildId